### DISCUSSION TITLE
Integrating Code Hike into docs.page

### DISCUSSION THREAD

**Discussion by cabljac** [code-hike/codehike#113] [Show and tell]
Hey, thought i'd post this here in case people are interested.

I maintain a docs generation app [docs.page](https://docs.page) which generates documentation from a github repo with minimal effort and configuration.

I was really impressed with Code Hike, and so i'm trying to integrate it (under an experimental option) into docs.page in our upcoming big release.

There are some teething issue as to be expected (scrollycoding crashes on hard refresh for example), but i think it will be really cool one it's all working!

**Comment by pomber** [comment]
wow! looks very good

**Reply by cabljac** [reply]
Thanks! It's a very rough work in progress so far.

**Reply by pomber** [reply]
Let me know the things that you wish would be easier to customize

**Reply by cabljac** [reply]
btw just realised i watched your 'build your own react' talk a while back, didn't realise you were one and the same! Great talk

**Comment by cabljac** [comment]
Essentially finished now! - 

 [here's the code hike demo stuff](https://docs.page/cabljac/test)

**Reply by cabljac** [reply]
I want to tweak the css a bit, but everything seems to work

**Reply by pomber** [reply]
amazing! 
let me know if there are some styles you think should be included on code hike side
